test/Coverity/ConfigFiles/FreeRTOSIPConfig.h

```diff
@@ -30,7 +30,7 @@
 #ifndef FREERTOS_IP_CONFIG_H
 #define FREERTOS_IP_CONFIG_H
 
-#define _static
+#define _static static
```

Comment:
I think the correct way is to remove `_static` from files - and replace it with `static`.

We had added that to make CBMC work - but it is not needed anymore.

Can we please do that?

Reply:
we use it for unit tests as well in some parts no ?